app/src/components/fields/CustomFieldArray.tsx

```diff
@@ -69,7 +67,7 @@ export interface CustomFieldArrayProps<ValueType = unknown> {
  * @param {*} props
  * @return {*}
  */
-const CustomFieldArray = <ValueType extends any>(props: CustomFieldArrayProps<ValueType>) => {
+const CustomFieldArray = <ValueType,>(props: CustomFieldArrayProps<ValueType>) => {
```

Comment:
nit: odd comma?

Reply:
The comma is needed; Because it's a .tsx file, the compiler thinks the angle tags are a React element and not a type specifier. We can alternatively use <Value extends any> instead of <Value,>; But we have to be more specific than "any" because "extend any" is a codesmell

Reply:
Gotcha, read a bit on it, and yeah makes sense. I wonder if we can just extend unknown instead? Would be nice if that works, as its a bit less obscure then the comma workaround.

Also, is that component ever used anywhere? I cant find any references to it.

Reply:
This component isn't currently used, no. I believe it was copied from BioHub (or maybe the other way around?). I'll try `extends unknown`.